cmd/lncli/commands.go

```diff
@@ -1289,6 +1289,61 @@ func unlock(ctx *cli.Context) error {
 	return nil
 }
 
+var changePasswordCommand = cli.Command{
+	Name: "changepassword",
+	Description: `
+	Change the encrypted wallet's password. It must be unlocked before
+	proceeding.
+
+	If you did not specify a password for your wallet (e.g. running lnd with
+	--noencryptwallet), be sure to leave the "current password" field empty.
+	`,
+	Action: actionDecorator(changePassword),
+}
+
+func changePassword(ctx *cli.Context) error {
+	ctxb := context.Background()
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+
+	fmt.Printf("Input current wallet password: ")
+	current, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		return err
+	}
+	fmt.Println()
+
+	fmt.Printf("Input new wallet password: ")
+	new, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		return err
+	}
+	fmt.Println()
+
+	fmt.Printf("Confirm new wallet password: ")
+	confirm, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		return err
+	}
+	fmt.Println()
+
+	if !bytes.Equal(new, confirm) {
+		return fmt.Errorf("passwords don't match")
+	}
+
+	req := &lnrpc.ChangePasswordRequest{
+		CurrentPassword: current,
+		NewPassword:     new,
```

Comment:
nit: use variable name that isn't a keyword

Reply:
Fixed.